src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/BaselineHelper.cs

```diff
@@ -89,27 +89,32 @@ public static string DiffFiles(string file1Path, string file2Path, ITestOutputHe
 
         public static string GetBaselineFilePath(string baselineFileName) => Path.Combine(GetAssetsDirectory(), "baselines", baselineFileName);
 
-        public static string RemoveNetTfmPaths(string source)
-        {
-            string pathSeparator = Regex.Escape(Path.DirectorySeparatorChar.ToString());
-            Regex netTfmRegex = new($"{pathSeparator}net[1-9]+\\.[0-9]+{pathSeparator}");
-            return netTfmRegex.Replace(source, $"{Path.DirectorySeparatorChar}{NetTfmPlaceholder}{Path.DirectorySeparatorChar}");
-        }
-
         public static string RemoveRids(string diff, bool isPortable = false) =>
             isPortable ? diff.Replace(Config.PortableRid, "portable-rid") : diff.Replace(Config.TargetRid, "banana-rid");
 
         public static string RemoveVersions(string source)
         {
+            // Remove version numbers for examples like "roslyn4.1", "net8.0", and "netstandard2.1".
+            string pathSeparator = Regex.Escape(Path.DirectorySeparatorChar.ToString());
+            string result = Regex.Replace(source, $@"{pathSeparator}(net|netstandard|roslyn)[1-9]+\.[0-9]+{pathSeparator}", match =>
```

Comment:
I don't think we should be replacing netstandard tfms.  Their usage doesn't float like net8.0 would.
```suggestion
            string result = Regex.Replace(source, $@"{pathSeparator}(net|roslyn)[1-9]+\.[0-9]+{pathSeparator}", match =>
```

Reply:
Addressed in [this commit](https://github.com/dotnet/installer/pull/17233/commits/8ec128e7e0e58aa70e87b705d20616b29250bb04).